setup.cfg

```diff
@@ -28,6 +28,7 @@ zip_safe = False
 install_requires =
     # Invenio core modules
     invenio-app>=1.3.4,<1.4.0
+    invenio-db[postgresql,mysql]>=1.0.14,<2.0.0
```

Comment:
won't this install both dbs? also, seems like versioning is missing?

Reply:
This will install only the drivers/packages, we do this everywhere now.
Versioning was optional, it is now [installed by default](https://github.com/inveniosoftware/invenio-db/blob/master/setup.cfg#L37).